stdlib/Statistics/src/Statistics.jl

```diff
@@ -821,8 +821,13 @@ function quantile!(q::AbstractArray, v::AbstractVector, p::AbstractArray;
     return q
 end
 
-quantile!(v::AbstractVector, p::AbstractArray; sorted::Bool=false) =
-    quantile!(similar(p,float(eltype(v))), v, p; sorted=sorted)
+function quantile!(v::AbstractVector, p::AbstractArray; sorted::Bool=false)
```

Comment:
Actually can't we just use this method for tuple `p`? `map` should do the right thing in both cases.

Reply:
Right - I will merge them.